modules/install-sno-generate-the-discovery-iso-manually.adoc

```diff
@@ -0,0 +1,150 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso-manually_{context}"]
+= Generate the discovery ISO manually
+
+Installing {product-title} on a single node requires a discovery ISO, which you can generate with manually with the following procedure.
+
+.Procedure
+
+. Retrieve the {product-title} client (`oc`):
```

Comment:
```suggestion
. Download the {product-title} client (`oc`) and make it available for use.
```

Reply:
@aireilly The style guide requires a colon when a CLI example follows.